endpoints/events/event_test.go

```diff
@@ -0,0 +1,591 @@
+package events
+
+import (
+	"context"
+	"encoding/base64"
+	"encoding/json"
+	"github.com/prebid/prebid-server/analytics"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/stored_requests"
+	"github.com/stretchr/testify/assert"
+	"io/ioutil"
+	"net/http/httptest"
+	"strings"
+	"testing"
+)
+
+// Mock Analytics Module
+type eventsMockAnalyticsModule struct {
```

Comment:
This is fine. For the future, a preferred approach would be to use the `github.com/stretchr/testify/mock` package. End result is the same.